test/unit/mdc-list/mdc-list.test.js

```diff
@@ -115,6 +190,33 @@ test('wrapFocus calls setWrapFocus on foundation', () => {
   td.verify(mockFoundation.setWrapFocus(true), {times: 1});
 });
 
+test('singleSelection true sets the click handler from the root element', () => {
+  const {root, component, mockFoundation} = setupTest();
+  component.singleSelection = true;
+  domEvents.emit(root, 'click');
+  td.verify(mockFoundation.handleClick(td.matchers.anything()), {times: 1});
+});
+
+test('singleSelection false removes the click handler from the root element', () => {
+  const {root, component, mockFoundation} = setupTest();
+  component.singleSelection = true;
+  component.singleSelection = false;
+  domEvents.emit(root, 'click');
+  td.verify(mockFoundation.handleClick(td.matchers.anything()), {times: 0});
+});
+
+test('singleSelection calls foundation setSingleSelection with the provided value', () => {
+  const {component, mockFoundation} = setupTest();
+  component.singleSelection = true;
+  td.verify(mockFoundation.setSingleSelection(true), {times: 1});
+});
+
+test('selectedIndex calls setSelectedIndex on foundation', () => {
```

Comment:
setSelectedIndex is a pretty complex function, but only has this one test. Might want to add more

Reply:
`setSelectedIndex` has nearly 100% coverage through the `click`/`key` handler tests.